documentation/docs/reference/software/interfaces/mqtt.md

```diff
@@ -59,171 +109,390 @@ Control of the focus stage. The message is a JSON object, speed is optional:
+    "force": False,
+    "recursive": True,
+    "ecotaxa": True,
+    "keep": True
+  }
 }
 ```
+The `action` element is the only element required. If no `path` is supplied, the whole images repository is segmented recursively (this is very long!).
+
+| Parameter     | Type   | Accepted Values      | Description                                            |
+|---------------|--------|----------------------|--------------------------------------------------------|
+| `path`        | string | path/to/directory    | Path to the directory to segment.                      |
+| `force`       | bool   | true, false          | Force re-segmentation even if previously done. It will overcome the presence of the file `done` that prevents resegmenting a folder already segmented. |
+| `recursive`   | bool   | true, false          | Process directories recursively, forcing parsing all folders below `path`. |
+| `ecotaxa`     | bool   | true, false          | Export an ecotaxa compatible archive.                  |
+| `keep`        | bool   | true, false          | Keep ROI files during ecotaxa export. It has no effect if not exporting to ecotaxa; the ROI files are kept by default. |
+
+ **Status/Error Messages related to the segment action**
+
+| Status/Error message | Description                                    |
+|-------------------|------------------------------------------------|
+| `Ready`              | The segmenter process has started and is ready. |
+| `Started`            | The segmentation process has begun. |
+| `Calculating flat`   | The frame background is being calculated. |
+| `Segmenting image %s, image %d/%d` | Segmentation of a specific image is in progress. |
+| `An exception was raised during the segmentation: %s.` | An error occurred during segmentation. |
+| `Done`               | The segmentation process is complete. |
+| `ERROR_INVALID_ACTION` | The specified action is invalid.              |
+| `ERROR_INVALID_PATH`   | The specified path is invalid or not found.    |
+| `ERROR_SEGMENTATION_FAILURE` | The segmentation process failed. 
+
+
+- **JSON message to `stop` segmentation**
+        ```json
+        {
+          "action": "stop"
+        }
+        ```
+**Required Parameters to `stop` segmentation**:
+
+| Parameter     | Type   | Accepted Values      | Description                                            |
+|---------------|--------|----------------------|--------------------------------------------------------|
+| `action`      | string | "stop"               | Specifies the action to stop segmentation.             |
+        
+ **Status/Error Messages related to the segment action**
+
+| Status/Error message | Description                                    |
+|-------------------|------------------------------------------------|
+| `Interrupted`        | The segmentation process was interrupted. |
+| `Busy`               | The segmenter is currently running and cannot update configurations. |
+
+
+## Common Log Errors
```

Comment:
As I had mentioned in a previous meeting, this section does not belong in the MQTT API page; it needs to be moved out to a separate page.